templates/kdump.conf

```diff
@@ -0,0 +1,13 @@
+# kdump configuration file
+
+{% if kdump_crash_path is defined -%}
+path {{ kdump_crash_path }}
+{%- endif %}
+
+{% if kdump_core_collector_args is defined -%}
+core_collector {{kdump_core_collector_args}}
```

Comment:
Your templating syntax is inconsistent. In other places in the role you have a space between `{{` and the variable name. Here and on line 12 you don't. Our preference is to have the spaces for the ease of reading, but at the least keep it consistent within a role.